oaff/app/oaff/app/data/sources/postgresql/stac_hybrid/postgresql_data_source.py

```diff
@@ -223,7 +223,7 @@ async def _get_derived_layers(self) -> Dict[str, PostgresqlLayer]:
                 bboxes=[table_spatial_extents[qualified_layer_name]],
                 intervals=table_temporal_extents[qualified_layer_name]
                 if qualified_layer_name in table_temporal_extents
-                else [[None, None]],
```

Comment:
The type of the `intervals` parameter is `List[List[Optional[str]]]`, so this should be `[[None, None]]`. I am not sure why you removed the enclosing list